src/public_info.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright 2017 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under (1) the MaidSafe.net Commercial License,
+// version 1.0 or later, or (2) The General Public License (GPL), version 3, depending on which
+// licence you accepted on initial access to the Software (the "Licences").
+//
+// By contributing code to the SAFE Network Software, or to this project generally, you agree to be
+// bound by the terms of the MaidSafe Contributor Agreement.  This, along with the Licenses can be
+// found in the root directory of this project at LICENSE, COPYING and CONTRIBUTOR.
+//
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.
+//
+// Please review the Licences for the specific language governing permissions and limitations
+// relating to use of the SAFE Network Software.
+
+// FIXME: remove when this module is finished
+#![allow(dead_code)]
```

Comment:
This shouldn't be needed now I guess?

Reply:
@Fraser999 this is left there because this module is not exported publicly (only crate-level, i.e. `pub(crate) use public_info;`), so this results in Clippy complaints about the dead code.

If you make it public, Clippy shows errors about the absent documentation. So we can either add documentation and make the module public, or leave this lint turned off in this module just for now.